content/post/060-ipfs-camp-keynotes-interviews.md

```diff
@@ -0,0 +1,61 @@
+
+Want to learn about how the IPFS Community and Ecosystem have been growing? [This video](https://www.youtube.com/watch?v=jpQnQbfhuBc&feature=youtu.be) from Molly Mackinlay, IPFS Project Lead, is a great overview of the scale, progress, and important milestones IPFS has achieved so far in 2019.
+
+### Closing
+On our last day, we celebrated the amazing learning, presentations, and courses with a [Space Training Program Graduation & Awards Ceremony](https://www.youtube.com/watch?v=INVQPepMMMU&feature=youtu.be). Who got the award for best interactive skit? Watch this to see the team!
+
+And finally, Juan set us back out through the airlock back toward Earth with his [IPFS Camp Sendoff](https://youtu.be/HDc7kEYHang), including a huge **THANK YOU** to everyone who made this camp possible - especially the tireless Protocol Labs Events Team who worked so hard to make this event a huge success!
+
+## Community Interviews
+IPFS Camp was all about sharing and forming connections between projects and individuals in the IPFS community, so we managed to get some sit-down time with a few wonderful community members to talk about their projects. Check out the videos below or skip right to the [playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTwwGv30nKdhr3GiIAnhbyE). 
+
+### Netflix Containers - Edgar Lee
+
+[![Screen Shot 2019-09-30 at 5 23 20 PM](https://user-images.githubusercontent.com/618519/65925910-890ec180-e3a7-11e9-8eb9-b632a4d3c146.png)](https://youtu.be/wNfk05D887M)
+
+In [this interview](https://youtu.be/wNfk05D887M), we heard from Edgar about how he's exploring distributing container layers through IPFS, and how it's changing his workflow at Netflix. <More Here>
+
+### Audius - Hareesh Nagaraj
+[![Screen Shot 2019-09-30 at 5 32 15 PM](https://user-images.githubusercontent.com/618519/65926083-439ec400-e3a8-11e9-9b5b-c40f118e3258.png)](https://youtu.be/3pSV6ai6QUI) 
+
+We spoke to Hareesh from [Audius](https://audius.co/), a decentralized music streaming platform, about how they're using IPFS as their primary file storing system. Check out their recent [launch blog post](https://techcrunch.com/2019/09/24/audius-music/) to learn more.
+
+### And More
+
+#### [Radicle & Monadic - Julian Arni](https://youtu.be/UCDHMKC8Seg) 
+Julian told us about his peer-to-peer collaboration platform Radicle and the benefits of decentralized systems. 
+
+#### [World Brain - Oliver Sauter](https://youtu.be/e1YfSa2_Drc) 
+We talked to Oliver about their decentralized search tool, Memex, and their peer-to-peer search engine for personal and collective knowledge, Worldbrain.io. 
+
+#### [IPFS Force - Steven Li](https://youtu.be/KsMqesFi4Ac) 
+And on top of all that, Steven joined us from Shanghai City to discuss his work on IPFS-Force.
+
+
+## That's All, Folks
```

Comment:
This is great Molly! Read through it all, and it's a pretty dope wrap up to the whole event! 

Should we find a way to host all of Yuni's amazing photos and do a share of those as well?

Reply:
Non-external link here: 
https://drive.google.com/open?id=1vL9tfkWqX1HMNiHhpdvkGVht4H5MjJsE

Reply:
I love that idea!! @yunigraham - IIRC you were going to have a selection you wanted to share out. I'm sure folks would love to flip through if we were to upload them somewhere!